### PR TITLE
Added tools for rss feed generation and other

### PR DESCRIPTION
**Pull request by dumprop**

### CODE REVIEW

File: pages/02.applications/04.wishlist/apps_wishlist.md [hunk 1]
```diff
@@ -236,6 +236,8 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [roadiz](https://www.roadiz.io) | Create a modern website |  | [Package Draft](https://github.com/YunoHost-Apps/roadiz_ynh) |
 | [rocketchat](https://rocket.chat) | A chat platform |  | [Package Draft](https://github.com/YunoHost-Apps/rocketchat_ynh) |
 | rs-short | An URL shortener | [Upstream](https://git.42l.fr/42l/rs-short) |  |
+| RSSHub | open source, easy to use, and extensible RSS feed generator. It's capable of generating RSS feeds from pretty much everything | [Upstream](https://github.com/DIYgod/RSSHub) |  |
+| rss-bridge | The RSS feed for websites missing it | [Upstream](https://github.com/RSS-Bridge/rss-bridge) |  |
```
Comment (alexAubin) [inline pages/02.applications/04.wishlist/apps_wishlist.md:240]: rss bridge is already available fyi

File: pages/02.applications/04.wishlist/apps_wishlist.md [hunk 2]
```diff
@@ -306,6 +314,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [upmpdcli](https://www.lesbonscomptes.com/upmpdcli/) |  | [Upstream](https://framagit.org/medoc92/upmpdcli) |  |
 | [Uwazi](https://www.uwazi.io/) |  | [Upstream](https://github.com/huridocs/uwazi) |  |
 | [Volumio](https://volumio.org) |  | [Upstream](https://github.com/volumio) |  |
+| [Vikunja](https://vikunja.io/) |  | [Upstream](https://kolaente.dev/vikunja/api) |  |
```
Comment (ericgaspar) [inline pages/02.applications/04.wishlist/apps_wishlist.md:317]: Vikunja can also be removed 😅 -> https://github.com/YunoHost-Apps/vikunja_ynh

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: pages/02.applications/04.wishlist/apps_wishlist.md (hunk 2 above)
Comment (dumprop) [reply pages/02.applications/04.wishlist/apps_wishlist.md]: wow, cool)

Comment (ericgaspar) [reply pages/02.applications/04.wishlist/apps_wishlist.md]: The package as still some issues...

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**